Makefile

```diff
@@ -175,7 +173,7 @@ LIBCALICO_VERSION?=$(shell git ls-remote git@github.com:projectcalico/libcalico-
 FELIX_REPO?=github.com/projectcalico/felix
 FELIX_VERSION?=$(shell git ls-remote git@github.com:projectcalico/felix master 2>/dev/null | cut -f 1)
 CONFD_REPO?=github.com/projectcalico/confd
-CONFD_VERSION?=$(shell git ls-remote git@github.com:projectcalico/confd master 2>/dev/null | cut -f 1)
+CONFD_VERSION?=$(shell git ls-remote git@github.com:projectcalico/confd feature-internal-rr 2>/dev/null | cut -f 1)
```

Comment:
Need to update this to master once merged.

Reply:
I meant to say once the confd PR was merged, which is has now, so we can update this :)